### PR TITLE
[cifuzz] Add timeout to http requests.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Fixes https://github.com/google/oss-fuzz/issues/9079

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
@halbecaf@mastodon.social

File: infra/cifuzz/http_utils.py [hunk 1]
```diff
@@ -81,10 +81,11 @@ def get_json_from_url(url):
   Returns:
     A dictionary deserialized from JSON or None on failure.
   """
-  response = requests.get(url)
+  response = requests.get(url, timeout=10)
   try:
```
Comment (kleisauke) [inline infra/cifuzz/http_utils.py:85]: I think `requests.get(url, timeout=10)` should be placed inside the `try`-block to ensure the `requests.exceptions.ReadTimeout` exception is caught.
```suggestion
  try:
    response = requests.get(url, timeout=10)
```

Comment (jonathanmetzman) [reply infra/cifuzz/http_utils.py]: Definitely. dumb mistake on my part!

Comment (jonathanmetzman) [reply infra/cifuzz/http_utils.py]: Thanks!

File: infra/cifuzz/http_utils.py [hunk 2]
```diff
@@ -81,10 +81,11 @@ def get_json_from_url(url):
   Returns:
     A dictionary deserialized from JSON or None on failure.
   """
-  response = requests.get(url)
   try:
+    response = requests.get(url, timeout=10)
```
Comment (oliverchang) [inline infra/cifuzz/http_utils.py:85]: nit: put 10 in a constant somewhere.

**Review by oliverchang — APPROVED** [review]

File: infra/cifuzz/http_utils.py (hunk 1 above)
Comment (kleisauke) [reply infra/cifuzz/http_utils.py]: A force push made the applied suggestion disappear, so this is still relevant at the time of writing.

Comment (kleisauke) [reply infra/cifuzz/http_utils.py]: Commit e408199d8bac2da6de39b14b7db36a18820413a8 looks good to me. :)

**Pull request merged**